src/cascadia/TerminalApp/Profile.cpp

```diff
@@ -690,18 +468,18 @@ bool Profile::IsDynamicProfileObject(const Json::Value& json)
 // - name: The name to generate a unique GUID from
 // Return Value:
 // - a uuidv5 GUID generated from the given name.
-GUID Profile::_GenerateGuidForProfile(const std::wstring& name, const std::optional<std::wstring>& source) noexcept
+winrt::guid Profile::_GenerateGuidForProfile(const hstring& name, const IReference<hstring>& source) noexcept

```

Comment:
i thought we _couldn't_ have an IReference to a string ......

Reply:
It's an IDL problem :/ fixed this though